daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1014,23 +1012,21 @@ object SBuiltin {
         coid,
         templateId,
         machine.lastLocation,
-        contextActors intersect stakeholders,
+        extraActors union (contextActors intersect stakeholders),
```

Comment:
I'd do this differently:
- if the `extraActors` are specified, then only put those there.
- If they are not specified, then use `contextActors intersect stakeholders`
As is now, non-maintainer stakeholders in the authorization context also become actors on the fetch node produced by a positive `lookupByKey`. So even non-signatories become actors and could have a say on this node. This feels unnatural.

Reply:
Yep, that's what I did. Haven't pushed yet.